src/toil/server/wes/abstract_backend.py

```diff
@@ -256,7 +256,8 @@ def collect_attachments(self, run_id: Optional[str], temp_dir: Optional[str]) ->
         else:
             raise MalformedRequestException("Missing 'workflow_url' in submission")
 
-        if "workflow_params" not in body:
-            raise MalformedRequestException("Missing 'workflow_params' in submission")
+        if "workflow_params" in body and not isinstance(body["workflow_params"], dict):
```

Comment:
Suggestion:  Could be shortened to:

```
if not isinstance(body.get("workflow_params"), dict):
```

Reply:
This doesn't actually work. If it's not there, we will `get` `None`, which isn't a `dict`, so we'll pass the `if`. We don't actually want to run the body when `workflow_params` isn't there.